packages/dds/map/src/test/mocha/directoryFuzzTests.spec.ts

```diff
@@ -111,7 +111,7 @@ describe("SharedDirectory fuzz", () => {
 		// Uncomment this line to replay a specific seed from its failure file:
 		// replay: 0,
 		saveFailures: { directory: dirPath.join(_dirname, "../../../src/test/mocha/results/2") },
-		skip: [63], // ! Fix tracked by AB#38522
+		skip: [],
```

Comment:
In addition to these fuzz tests, looks like you can also reenable a bunch of the seeds in `localServerStress.spec.ts` and `localServerStressOrderSequentially.spec.ts`

Reply:
Looks like there are still some issues with a few seeds, but I re-enabled what I could in latest